sway-lib-std/src/num.sw

```diff
@@ -0,0 +1,73 @@
+library num;
+
+impl u64 {
+    /// The smallest value that can be represented by this integer type.
+    pub fn MIN() -> u64 {
+        0
+    }
+
+    /// The largest value that can be represented by this integer type,
+    /// 2<sup>64</sup> - 1.
+    pub fn MAX() -> u64 {
+        18446744073709551615
+    }
+
+    /// The size of this integer type in bits.
+    pub fn BITS() -> u32 {
```

Comment:
Any reason the return type here is different than MIN and MAX? Ditto for each of the other types' BITS methods.

Reply:
That's just how Rust does it for whatever reason: https://doc.rust-lang.org/std/primitive.u64.html#associatedconstant.BITS

Reply:
I see . That's weird.
You call a method on a u64 and get a u32 back, seems like needless mental overhead. Must be some reason though...

Reply:
🤷. I vote for keeping it in line with Rust unless we have a reason not to. I'm sure they had their reasons.